ethereum/deployments.ts

```diff
@@ -17,8 +17,10 @@ export interface RigsDeployment {
   allowlistTable: string;
   partsTable: string;
   layersTable: string;
-  tokensTable: string;
   attributesTable: string;
+  lookupsTable: string;
+  pilotSessionsTable: string;
+  displayAttributes: boolean;
```

Comment:
This file needs to be updated once we publish the new Rigs schema to the various chains. #238